app/templates/components/builds-item.hbs

```diff
@@ -27,7 +27,7 @@
       </div>
     {{else}}
       <div class="row-message">
-        {{{format-message build.commit.message short="true" repo=build.repo eventType=build.eventType}}}
+        {{{format-message build.commit.message short="true" repo=build.repo eventType=build.eventType maxLength=50}}}
```

Comment:
Just curious, why is the `maxLength` value different in the 2 files? i.e. one is 60 and the other is 50. Thanks!

Reply:
Because the containers' widths are different, thus amount of text that appears is different too.

Reply:
Ah! Makes sense. Thanks for explaining @rafael-travis!

Reply:
No problem, any time @cotsog!